run-repeatedly.sh

```
@@ -48,68 +31,31 @@ check_snapshot_diffs() {
-    # Run the integration tests
     DOTNET_TEST_ARGS_ARRAY=("--no-restore" "--no-build" "test/Altinn.App.Integration.Tests/" "--logger" "console;verbosity=detailed")
     if [ -n "$TEST_FILTER" ]; then
         DOTNET_TEST_ARGS_ARRAY+=("--filter" "$TEST_FILTER")
     fi
     
-    if env TEST_LOCALTEST_BRANCH="$TEST_LOCALTEST_BRANCH" TEST_KEEP_CONTAINERS="$TEST_KEEP_CONTAINERS" dotnet test "${DOTNET_TEST_ARGS_ARRAY[@]}"; then
+    if dotnet test "${DOTNET_TEST_ARGS_ARRAY[@]}"; then
         echo "✓ Test run $i completed successfully"
 
-        # Check for snapshot differences
         if ! check_snapshot_diffs $i; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)